app/src/main/java/org/jellyfin/androidtv/util/KeyProcessor.java

```diff
@@ -221,7 +221,8 @@ public static boolean HandleKey(int key, BaseRowItem rowItem, Activity activity)
         return false;
     }
 
-    private static void createItemMenu(BaseRowItem rowItem, UserItemDataDto userData, Activity activity) {
+    // return the created PopupMenu so that the caller can dismiss it if needed
```

Comment:
This comment doesn't seem that necessary to me. Besides, it doesn't use Javadoc/Kdoc so it's not even useful in an IDE.